### PR TITLE
post: pandas 1.0

### PR DESCRIPTION
**Pull request by TomAugspurger**
cc @pandas-dev/pandas-core.

Will publish this when announcing 1.0

### CODE REVIEW

File: content/pandas-1.0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Reflections
+
+The world of scientific Python has changed a lot since pandas was started.  In 2011, [the ecosystem was fragmented](https://wesmckinney.com/blog/a-roadmap-for-rich-scientific-data-structures-in-python/): a standard *rich* data structure for statistics and data science had yet to emerge. This echos a similar story for NumPy, which consolidated array efforts that were [previously fragmented](https://numpy.org/old_array_packages.html).
+
+Over the subsequent years, pandas emerged as a *de facto* standard. It’s used by data scientists and analysts and as a data structure for other libraries to build on top of. StackOverflow [cited pandas](https://stackoverflow.blog/2017/09/14/python-growing-quickly/) as one of the reasons for Python being the fastest growing major programming language.
+
+![Growth of pandas](https://149351115.v2.pressablecdn.com/wp-content/uploads/2017/09/related_tags_over_time-1-1000x1000.png)
+
+Today, the ecosystem is in another phase of exploration.
```
Comment (TomAugspurger) [inline content/pandas-1.0.md:14]: This subsection is the only notable change from what I shared earlier.

File: content/pandas-1.0.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+Title: pandas 1.0
+Date: 2020-01-28
+
+Today pandas celebrates its 1.0.0 release. In many ways this is just a normal release with a host of new features, performance improvements, and bug fixes, which are documented in our [release notes](https://pandas.pydata.org/pandas-docs/version/1.0.0/whatsnew/v1.0.0.html). But it’s also something a bit more — a milestone for the project beyond just the commits. We wanted to take some time to reflect on where we've been and where we're going.
```
Comment (jorisvandenbossche) [inline content/pandas-1.0.md:4]:
```suggestion
Today pandas celebrates its 1.0.0 release. In many ways this is just a normal release with a host of new features, performance improvements, and bug fixes, which are documented in our [release notes](https://pandas.pydata.org/pandas-docs/version/1.0/whatsnew/v1.0.0.html). But it’s also something a bit more — a milestone for the project beyond just the commits. We wanted to take some time to reflect on where we've been and where we're going.
```

(if you ensure that link exists of course)

File: content/pandas-1.0.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+While a “1.0.0” version might seem arbitrary or anti-climactic (given that pandas as a codebase is nearly 12 years old), we see it as a symbolic milestone celebrating the growth of our core developer team and depth of our contributor base.  Few open source projects are ever truly “done” and pandas is no different. We recognize the essential role that pandas now occupies, and we intend to continue to evolve the project and adapt to the needs of the world’s data wranglers.
```
Comment (jorisvandenbossche) [inline content/pandas-1.0.md:22]: "anti-climactic" is not really clear to me (I also didn't directly know the word, so maybe it's my non-native speakerness)

File: content/pandas-1.0.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+While a “1.0.0” version might seem arbitrary or anti-climactic (given that pandas as a codebase is nearly 12 years old), we see it as a symbolic milestone celebrating the growth of our core developer team and depth of our contributor base.  Few open source projects are ever truly “done” and pandas is no different. We recognize the essential role that pandas now occupies, and we intend to continue to evolve the project and adapt to the needs of the world’s data wranglers.  
+
+# Going Forward
+
+Our [roadmap](https://pandas.pydata.org/pandas-docs/version/1.0.0/development/roadmap.html) contains an up-to-date listing of where we see the project heading over the next few years.
+Needless to say, there's still plenty to do.
+
+Check out the [release notes](https://pandas.pydata.org/pandas-docs/version/1.0.0/whatsnew/v1.0.0.html) and visit the [installation page](https://pandas.pydata.org/pandas-docs/version/1.0.0/getting_started/install.html) for instructions on updating to pandas 1.0.
```
Comment (jorisvandenbossche) [inline content/pandas-1.0.md:29]: Should we mention something specifically about updating? (what I put in a note in the whatsnew, about having removed a lot of deprecated things, so it is advised to first update to 0.25 if you didn't do that yet (if people agree with that advice of course))

File: content/pandas-1.0.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+While a “1.0.0” version might seem arbitrary or anti-climactic (given that pandas as a codebase is nearly 12 years old), we see it as a symbolic milestone celebrating the growth of our core developer team and depth of our contributor base.  Few open source projects are ever truly “done” and pandas is no different. We recognize the essential role that pandas now occupies, and we intend to continue to evolve the project and adapt to the needs of the world’s data wranglers.  
+
+# Going Forward
+
+Our [roadmap](https://pandas.pydata.org/pandas-docs/version/1.0.0/development/roadmap.html) contains an up-to-date listing of where we see the project heading over the next few years.
```
Comment (jorisvandenbossche) [inline content/pandas-1.0.md:26]: It's maybe not *that* up to date :) 
(that's a separate issue, though, that we should do an update of it)

**Review by jorisvandenbossche — COMMENTED** [review]
Thanks a lot! I think it is good to keep it rather short as it is now

Added a few tiny comments.

File: content/pandas-1.0.md (hunk 3 above)
Comment (TomAugspurger) [reply content/pandas-1.0.md:22]: I think it's OK here.

File: content/pandas-1.0.md (hunk 4 above)
Comment (TomAugspurger) [reply content/pandas-1.0.md:29]: I'm ok with omitting that, since it's the first note in the release notes we just linked to.

**Review by jreback — APPROVED** [review]
very nice @TomAugspurger strikes a good balance on historical and go forward perspectives plans and objectives

ship it!

File: content/pandas-1.0.md (hunk 3 above)
Comment (jbrockmendel) [inline content/pandas-1.0.md:22]: does markdown prefer these fancy quotation marks vs the ascii variant?

**Comment by TomAugspurger** [comment]
Shouldn't matter.

**Comment by jbrockmendel** [comment]
looks nice, thanks for putting this together

**Pull request merged**